src/main/webapp/WEB-INF/pages/admin/user.ftl

```diff
@@ -79,40 +79,10 @@
                         });
                         $('#user\\.grantedAccessTo').multiSelect('select', [
                             <#if user.grantedAccessTo?has_content >
-                                    ${ "'"+user.grantedAccessTo?split(", ")?join("', '")+"'" } 
-                                </#if>
-                            ]);  	
-                            
-                            <#if "${newUser!}"=="yes">
-                                $("input#user\\.email").on( "focusout", function(){
-                                if ($("input#user\\.email").val().endsWith("@humboldt.org.co")){
-                                    // Remove those restricted resources that are actually not restricted for @humboldt users 
-                                    <#if restrictedResourcesForAllButIAvHUsers?has_content>
-                                        <#list restrictedResourcesForAllButIAvHUsers as rR>		
-                                                $('#user\\.grantedAccessTo').find("option[value='${rR.shortname}']").remove();
-                                                $('#ms-user\\.grantedAccessTo').find("ul.ms-list li").filter(function() {
-                                                    return $(this).text() == "${rR.shortname}";
-                                                }).remove();  
-                                            </#list>
-                                        </#if>
-                                } else { // The user to be created is not @humboldt.org.co, so refresh restricted resources lists
-                                    <#if restrictedResourcesForAllButIAvHUsers?has_content || restrictedResourcesForIAvHUsers?has_content >
```

Comment:
I think this is the logic that manages the list at the left side square this need to be evaluated because all the resources does not load just the ones restrictedResourcesForAllButIAvHUsers if the user is @humboldt.org then the resources does not load.

Reply:
I removed this because seems to be shown when there is a new user, and this behaviour never happens.